docs/Example Schedules/CE Schedules/compe-kteal2.md

```diff
@@ -0,0 +1,180 @@
+---
+tags:
+- Robotics
+- Artificial intelligence and machine learning
+---
+# Robotics, Statistics Minor, 8 Semesters
+
+Hi, I'm currently a second-semester Junior (CE '25) also pursuing a Statistics minor. I'm focused on software, robotics, autonomy, and controls. Originally, I had planned to use one semester as a co-op but decided to take more technical electives instead. My schedule is definitely on the heavy side but it allows you to spend most of Junior and Senior year taking tech electives that you're interested in, while never having to double up on the big 4 ECE classes.
+
+## Features of this Schedule
+
+- Can be easily modified to 7 semesters (or less)
+- No summer or winter classes, so internship options every summer
+- Covers most of the required classes for MEng in Robotics and Autonomy in undergrad
+- Statistics minor + James Scholar
+
+## Schedule
+
+### AP Credits and Proficiency Tests
+| Course Name | Credit Hours |
+| --- | --- |
+| [CHEM102](../../Course%20Wiki/Other%20Course%20Offerings/CHEM102.md) | 3 |
+| [CHEM104](../../Course%20Wiki/Other%20Course%20Offerings/CHEM104.md) | 3 |
+| [CS101](../../Course%20Wiki/CS%20Course%20Offerings/CS101.md) | 3 |
+| ECON102 | 3 |
+| ECON103 | 3 |
+| IB150 | 4 |
+| [MATH220](../../Course%20Wiki/MATH%20Course%20Offerings/MATH220.md) | 5 |
+| [MATH231](../../Course%20Wiki/MATH%20Course%20Offerings/MATH231.md) | 3 |
+| MCB150 | 4 |
+| [PHYS211](../../Course%20Wiki/PHYS%20Course%20Offerings/PHYS211.md) | 4 |
+| PS101 | 3 |
+| RHET105 | 4 |
+| **Total** | **42** |
```

Comment:
For all courses, can you also add the title of the course?  So, format as: CHEM102 - General Chemistry I

Reply:
do you want me to include the course title in the hyperlink or leave it as plaintext

Reply:
> do you want me to include the course title in the hyperlink or leave it as plaintext

Include the course title in the hyperlink.